src/style.css

```diff
@@ -0,0 +1,69 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  height: 60px;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo-link {
+  margin: auto 0;
+  margin-right: 117px;
+}
+
+.nav-list-item {
+  display: inline-flex;
+  margin: 0 10px;
```

Comment:
add only margin-right to all but the last
![image](https://user-images.githubusercontent.com/63306833/130020860-cf843774-86b1-401c-b78a-58810c50c5ce.png)